### DISCUSSION TITLE
Info O2 di semarang

### DISCUSSION THREAD

**Discussion by marthatyas** [kawalcovid19/wargabantuwarga.com#79] [Show and tell]
Terkait info pengisian ulang O2 di semarang. Agen a.n Pak Yan, nomornya tidak bisa dihubungi. klo admin ada waktu, mungkin bisa dihapus atau direvisi. 
Terima kasih atas kerja kerasnya!

**Comment by zainfathoni** [comment]
Terima kasih atas laporannya, sudah kami teruskan ke tim terkait untuk ditindaklanjuti. 🙏
